src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/client/WorkforceAllocationsToDeliusApiClient.kt

```diff
@@ -34,19 +32,23 @@ class WorkforceAllocationsToDeliusApiClient(private val webClient: WebClient) {
       }
   }
 
-  fun getSummaryByCrn(crn: String): Mono<PersonSummary> {
+  /**
+   * Returns the Person Summary from person-resources/workforce api
+   * @param crnOrNoms: CRN or NOMS
+   */
+  fun getSummaryByCrn(crnOrNoms: String): Mono<PersonSummary> {
```

Comment:
I think for by Noms you need to supply a query `type=NOMS` parameter rather than it doing either or in the endpoint? https://workforce-allocations-to-delius-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html#/person-resource/findPerson